files/install_python.sh

```diff
@@ -0,0 +1,31 @@
+#/bin/bash
+
+set -e
+
+if [ -d /opt/python ]; then
+  # let's clear any old install
+  rm /opt/python -rf
```

Comment:
@tyranron 
I'm doing this so that when this runs due to the versions changing we clear everything (including pip) and do a fresh install.